src/drf_pydantic/base_model.py

```diff
@@ -50,3 +50,31 @@ def __new__(
 class BaseModel(pydantic.BaseModel, metaclass=ModelMetaclass):
     # Populated by the metaclass or manually set by the user
     drf_serializer: ClassVar[type[serializers.Serializer]]
+
+    @classmethod
+    def model_rebuild(
+        cls,
+        *,
+        force: bool = False,
+        raise_errors: bool = True,
+        _parent_namespace_depth: int = 2,
+        _types_namespace: dict[str, Any] | None = None,
+    ) -> bool | None:
+        ret = super().model_rebuild(
+            force=force,
+            raise_errors=raise_errors,
+            _parent_namespace_depth=_parent_namespace_depth,
+            _types_namespace=_types_namespace,
+        )
+
+        if not hasattr(cls, "drf_serializer") or getattr(cls, "drf_serializer") in (
+            getattr(base, "drf_serializer", None) for base in cls.__mro__[1:]
+        ):
+            SERIALIZER_REGISTRY.pop(cls)
+            setattr(
+                cls,
+                "drf_serializer",
+                create_serializer_from_model(cls),
+            )
```

Comment:
can be replaced with
```suggestion
            cls.drf_serializer = create_serializer_from_model(cls)
```
since we know the attribute's name